### DISCUSSION TITLE
Low Productivity Streams and ER/GPP modification

### DISCUSSION THREAD

**Discussion by NicoleGotkowski** [USGS-R/streamMetabolizer#419] [Q&A]
Hi everyone,

I am currently using streamMetabolizer to measure low-productivity streams (High Arctic).  I have set my priors to other Arctic streams (GPP_daily_mu=0.029, GPP_daily_sigma=1, ER_daily_mu = -1.87, ER_daily_sigma = 1). However, in some of my streams, I cannot achieve biologically possible results (all positive ER values) at any point throughout the ~40 day study period. These tend to be the wider streams in the sub-catchment. To remedy this, I have set a requirement that ER <= 0, and GPP >=0. This gives me pretty similar results in every way, except the values of ER are biologically possible. The rhat values end up being less than or equal to 1.1 and ER and K600 don't end up being significantly correlated.

However, I am able to successfully achieve biologically possible days for some of my other streams without the requirement that ER <= 0, and GPP >=0. Some of those sites have better rhat values and less correlation between ER and K600 without any modification at all.

For my study, I am looking at the correlation between environmental variables (nutrients, hydrology, etc...) and daily rates of GPP and ER. I was wondering if there would be anything wrong with comparing the values of these streams, even though some I have the ER and GPP requirements whereas others I do not.  Would this create any kind of bias or confounding variables? 

Thanks for any help,

Nicole

**Comment by robohall** [comment]
Nicole,

I would need more info to properly diagnose, but a few things pop up.


to measure low-productivity streams

1. sM needs GPP to kick O2 out or equilibrium.  The return to equilibrium at night imparts a shape on the curve from which it can estimate K (the O2 curve also gets displaced downtime relative to solar noon).  Low GPP implies low DO swings and low ability to estimate K.  Is that the case?  How large is the diel excursion in terms of %DO sat?

2.  It is the Arctic so always light, is there enough dark tyo measure this change in DO?

3.  positive ER implies DO that always exceeds saturation.  That can happen in rivers with big spillways and rapids, and in lakes where NEP is positive and K is low.  It might also happen in rivers that never go dark.  Are DO data always positive?

Bob


On Feb 21, 2023, at 8:16 AM, Nicole Gotkowski ***@***.******@***.***>> wrote:



Hi everyone,

I am currently using streamMetabolizer to measure low-productivity streams (High Arctic). I have set my priors to other Arctic streams (GPP_daily_mu=0.029, GPP_daily_sigma=1, ER_daily_mu = -1.87, ER_daily_sigma = 1). However, in some of my streams, I cannot achieve biologically possible results (all positive ER values) at any point throughout the ~40 day study period. These tend to be the wider streams in the sub-catchment. To remedy this, I have set a requirement that ER <= 0, and GPP >=0. This gives me pretty similar results in every way, except the values of ER are biologically possible. The rhat values end up being less than or equal to 1.1 and ER and K600 don't end up being significantly correlated.

However, I am able to successfully achieve biologically possible days for some of my other streams without the requirement that ER <= 0, and GPP >=0. Some of those sites have better rhat values and less correlation between ER and K600 without any modification at all.

For my study, I am looking at the correlation between environmental variables (nutrients, hydrology, etc...) and daily rates of GPP and ER. I was wondering if there would be anything wrong with comparing the values of these streams, even though some I have the ER and GPP requirements whereas others I do not. Would this create any kind of bias or confounding variables?

Thanks for any help,

Nicole

**Comment by NicoleGotkowski** [comment]
Hi Bob,

I am definitely seeing the impact of daily variation in light levels on DO curves. The Arctic does experience days towards August where PAR=0, and I have ~14 days where this is the case. Even in days where PAR does not =0 at all, there is still a distinct curve. This leads me to believe there is enough darkness. I have even tried only including only days where there is 0 darkness, and there doesn't seem to be much change in the results.

Many of these streams are always highly saturated, and DO is always positive with the exception of random data points that are likley just logger errors. 

I had assumed this was because K was generally what was controlling oxygen saturation, but there are definitely daily curves. There are no spillways, but there are rapids. 

I have included oxygen/light curves in the file below for reference.

Thanks,

Nicole



[Oxygen_Curves_GitHub.docx](https://github.com/USGS-R/streamMetabolizer/files/10795552/Oxygen_Curves_GitHub.docx)

**Comment by robohall** [comment]
Yes, you have huge diel swings, these rivers may not be such low productivity!.  O2 does not go < 1000, that is concerning.  How did you estimate DO%sat?  Calculate that vis stream metabolizer and not via sensor output they can be wrong, (e.g. YSI algorithm is flawed).  How were sensors calibrated?  A quick check of  your osat shows they look about right for sea level.

what is gas exchange estimates?  GPP?

Did CB04 go dry?

On Feb 21, 2023, at 9:30 AM, Nicole Gotkowski ***@***.******@***.***>> wrote:



Hi Bob,

I am definitely seeing the impact of daily variation in light levels on DO curves. The Arctic does experience days towards August where PAR=0, and I have ~14 days where this is the case. Even in days where PAR does not =0 at all, there is still a distinct curve. This leads me to believe there is enough darkness. I have even tried only including only days where there is 0 darkness, and there doesn't seem to be much change in the results.

Many of these streams are always highly saturated, and DO is always positive with the exception of random data points that are likley just logger errors.

I had assumed this was because K was generally what was controlling oxygen saturation, but there are definitely daily cruves. There are no spillways, but there are rapids.

I have included oxygen/light curves in the file below for reference.

Thanks,

Nicole

Oxygen_Curves_GitHub.docx<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUSGS-R%2FstreamMetabolizer%2Ffiles%2F10795552%2FOxygen_Curves_GitHub.docx&data=05%7C01%7Cbob.hall%40flbs.umt.edu%7Cfb4fe935ecfd41bc54ef08db1428f37c%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C638125939876709210%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=tO7f%2F38SIDnY%2Fj0gNATlO7Oq4vC4YGhlldxZFnKyaRA%3D&reserved=0>

**Reply by woodwardb1** [reply]
> Yes, you have huge diel swings, these rivers may not be such low productivity!. O2 does not go < 1000, that is concerning. How did you estimate DO%sat? Calculate that vis stream metabolizer and not via sensor output they can be wrong, (e.g. YSI algorithm is flawed). How were sensors calibrated? A quick check of your osat shows they look about right for sea level. what is gas exchange estimates? GPP? Did CB04 go dry? On Feb 21, 2023, at 9:30 AM, Nicole Gotkowski ***@***.******@***.***>> wrote: Hi Bob, I am definitely seeing the impact of daily variation in light levels on DO curves. The Arctic does experience days towards August where PAR=0, and I have ~14 days where this is the case. Even in days where PAR does not =0 at all, there is still a distinct curve. This leads me to believe there is enough darkness. I have even tried only including only days where there is 0 darkness, and there doesn't seem to be much change in the results. Many of these streams are always highly saturated, and DO is always positive with the exception of random data points that are likley just logger errors. I had assumed this was because K was generally what was controlling oxygen saturation, but there are definitely daily cruves. There are no spillways, but there are rapids. I have included oxygen/light curves in the file below for reference. Thanks, Nicole Oxygen_Curves_GitHub.docx<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUSGS-R%2FstreamMetabolizer%2Ffiles%2F10795552%2FOxygen_Curves_GitHub.docx&data=05%7C01%7Cbob.hall%40flbs.umt.edu%7Cfb4fe935ecfd41bc54ef08db1428f37c%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C638125939876709210%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=tO7f%2F38SIDnY%2Fj0gNATlO7Oq4vC4YGhlldxZFnKyaRA%3D&reserved=0> — Reply to this email directly, view it on GitHub<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUSGS-R%2FstreamMetabolizer%2Fdiscussions%2F419%23discussioncomment-5066808&data=05%7C01%7Cbob.hall%40flbs.umt.edu%7Cfb4fe935ecfd41bc54ef08db1428f37c%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C638125939876709210%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=qpdGICa2XUIY5UxTngO1ClI6DzvJYTY7njR%2FiHTOzf0%3D&reserved=0>, or unsubscribe<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAC4CU5WDDCWRQHSQIYHA6MDWYTUSNANCNFSM6AAAAAAVDFSZKQ&data=05%7C01%7Cbob.hall%40flbs.umt.edu%7Cfb4fe935ecfd41bc54ef08db1428f37c%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C638125939876709210%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=2j6S0CS%2BQ1hI6x8UM6c62cng2wtWhAU%2BkyJ%2B7ONKjHE%3D&reserved=0>. You are receiving this because you commented.Message ID: ***@***.***>

Hi Bob, 

Can you provide anymore information about the flaw in the YSI %sat algorithm?  We used these all of the time and are always looking to assess the data they provide.  

Cheers,
Ben

**Comment by NicoleGotkowski** [comment]
Yes, CB04 did go dry. 

Some sites I had did go below 100% saturation. I did not include all of my streams as I had 16. I will have to double check the DO.sat calculation and the calibration, as I did not due it myself.  

I have attached my resulting values of GPP and K600 in the following attachment (It includes the average over the study period, and both the modified and unmodified models).


[streamMetabolizer_Results_GitHub.xlsx](https://github.com/USGS-R/streamMetabolizer/files/10796677/streamMetabolizer_Results_GitHub.xlsx)

**Comment by NicoleGotkowski** [comment]
Hi Bob,

The D-Opto loggers were calibrated by comparing a known concentration of DO (fully saturated) to a known concentration (0% saturation - tap water and sodium sulfite). This is the standard method for the D-Opto Logger Dissolved Oxygen Logger by Zebra-tech Ltd. When calculating with the calc_DO_sat I get the exact same results. Is it possible these streams are very productive and have high reaeration rates?

Thanks for your thoughts,
Nicole

**Comment by robohall** [comment]
Ben,

Yes.  They record % sat at sea level and not % sat corrected for elevation for elevation.  So if you are in Denver Colorado, the % sat will be 83%.  If you then  estimate oxygen saturation as [DO] / 0.83 and import this value into streamMetabolizer (or do anything else with it) then you are in trouble as your estimated O2  sats are way too high. To be fair someone working in Denver would figure this out, but someone 200 m above sea level would not, and they wold over estimate ER by a lot.

Bob

On Mar 13, 2023, at 8:18 PM, woodwardb1 ***@***.******@***.***>> wrote:



Yes, you have huge diel swings, these rivers may not be such low productivity!. O2 does not go < 1000, that is concerning. How did you estimate DO%sat? Calculate that vis stream metabolizer and not via sensor output they can be wrong, (e.g. YSI algorithm is flawed). How were sensors calibrated? A quick check of your osat shows they look about right for sea level. what is gas exchange estimates? GPP? Did CB04 go dry? On Feb 21, 2023, at 9:30 AM, Nicole Gotkowski @.@.>> wrote: Hi Bob, I am definitely seeing the impact of daily variation in light levels on DO curves. The Arctic does experience days towards August where PAR=0, and I have ~14 days where this is the case. Even in days where PAR does not =0 at all, there is still a distinct curve. This leads me to believe there is enough darkness. I have even tried only including only days where there is 0 darkness, and there doesn't seem to be much change in the results. Many of these streams are always highly saturated, and DO is always positive with the exception of random data points that are likley just logger errors. I had assumed this was because K was generally what was controlling oxygen saturation, but there are definitely daily cruves. There are no spillways, but there are rapids. I have included oxygen/light curves in the file below for reference. Thanks, Nicole Oxygen_Curves_GitHub.docxhttps://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUSGS-R%2FstreamMetabolizer%2Ffiles%2F10795552%2FOxygen_Curves_GitHub.docx&data=05%7C01%7Cbob.hall%40flbs.umt.edu%7Cfb4fe935ecfd41bc54ef08db1428f37c%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C638125939876709210%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=tO7f%2F38SIDnY%2Fj0gNATlO7Oq4vC4YGhlldxZFnKyaRA%3D&reserved=0<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUSGS-R%2FstreamMetabolizer%2Ffiles%2F10795552%2FOxygen_Curves_GitHub.docx&data=05%7C01%7Cbob.hall%40flbs.umt.edu%7C61693f7689f948e2145008db24326fb2%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C638143572206052753%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=yFugafUTIzIQgIDqKPkspMpOZuUP8pl29xJ2Pq3fSnM%3D&reserved=0> — Reply to this email directly, view it on GitHubhttps://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUSGS-R%2FstreamMetabolizer%2Fdiscussions%2F419%23discussioncomment-5066808&data=05%7C01%7Cbob.hall%40flbs.umt.edu%7Cfb4fe935ecfd41bc54ef08db1428f37c%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C638125939876709210%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=qpdGICa2XUIY5UxTngO1ClI6DzvJYTY7njR%2FiHTOzf0%3D&reserved=0<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUSGS-R%2FstreamMetabolizer%2Fdiscussions%2F419%23discussioncomment-5066808&data=05%7C01%7Cbob.hall%40flbs.umt.edu%7C61693f7689f948e2145008db24326fb2%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C638143572206052753%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=XkK%2B2DhTn78%2Fzjbtf4n%2B3Sg%2B5zhszlroTOiuaUoHjcE%3D&reserved=0>, or unsubscribehttps://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAC4CU5WDDCWRQHSQIYHA6MDWYTUSNANCNFSM6AAAAAAVDFSZKQ&data=05%7C01%7Cbob.hall%40flbs.umt.edu%7Cfb4fe935ecfd41bc54ef08db1428f37c%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C638125939876709210%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=2j6S0CS%2BQ1hI6x8UM6c62cng2wtWhAU%2BkyJ%2B7ONKjHE%3D&reserved=0<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAC4CU5WDDCWRQHSQIYHA6MDWYTUSNANCNFSM6AAAAAAVDFSZKQ&data=05%7C01%7Cbob.hall%40flbs.umt.edu%7C61693f7689f948e2145008db24326fb2%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C638143572206052753%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=CkTmId6MVHihh0kQhQr4X%2BOA7sn9d5LA2ezRP3yDPBw%3D&reserved=0>. You are receiving this because you commented.Message ID: @.***>

Hi Bob,

Can you provide anymore information about the flaw in the YSI %sat algorithm? We used these all of the time and are always looking to assess the data they provide.

Cheers,
Ben

**Reply by woodwardb1** [reply]
Thanks Bob, much appreciated!
